transport-native-epoll/src/main/java/io/netty/channel/epoll/Native.java

```diff
@@ -196,7 +196,8 @@ private static void loadNativeLibrary() {
         if (!name.startsWith("linux")) {
```

Comment:
Should this be replaced with calls to PlatformDependent to get this information? And missing ones added there?

Reply:
maybe... I did want to keep the change focused ;)